First pull of 2D Brownian Dynamics

aaguirre-rdit opened the pull request.

aaguirre-rdit committed:
Create brownlan_dynamics_2D

aaguirre-rdit committed:
Delete brownlan_dynamics_2D

aaguirre-rdit committed:
Create t

aaguirre-rdit committed:
Create t

aaguirre-rdit committed:
Uploaded simulation data

aaguirre-rdit committed:
Delete t

aaguirre-rdit committed:
Delete t

aaguirre-rdit committed:
Ready 1st pull.

Documented and uploaded ipynb. Now ready for first pull request.

daviddesancho reviewed (changes requested):
Great work! Just a few changes to incorporate.
(1) At the top of the notebook may be worth mentioning where the potential comes from and discarding the explanation on the diffusion coefficients. Just say "isotropic diffusion" on a 2D potential, because the potential user does not have to know anything about smFS.
(2) In [3] and in general for time series data, make axis start from 0 and end wherever the dataset ends. 
(3) In [27] change y axis from 1 to 9 to highlight differences in barrier.
(4) You can add subsections to the notebook using "###" so that we explain first the trajectory analysis, the PMF, the assignment, the estimation...
(5) Right after [17] ¿Can you plot the first eigenvector by mapping the state indexes on the 2D grid? We should see that the two wells are perfectly separated.

aaguirre-rdit committed:
Changes done

Changes made:
- Added sections.
- Changed plot limits.
- Mapped right eigenvector on 2d grid.
